schemas/livePaperResourceItem.schema.tpl.json

```diff
@@ -0,0 +1,27 @@
+{
+    "_type": "https://openminds.ebrains.eu/publications/LivePaperResourceItem",
+    "required": ["isPartOf", "resourceType", "name", "iri"],
```

Comment:
formally items should be listed in seperate lines

Reply:
done